pandas/tests/frame/methods/test_reset_index.py

```diff
@@ -50,6 +50,7 @@ def test_set_index_reset_index_dt64tz(self):
         df = result.set_index("foo")
         tm.assert_index_equal(df.index, idx)
 
+    @pytest.mark.filterwarnings("ignore:Timestamp.freq is deprecated:FutureWarning")
```

Comment:
What's causing this test to raise a warning?

Reply:
looks like it doesnt after one of the more recent comments, good catch,  will remove